### DISCUSSION TITLE
architect/actions

### DISCUSSION THREAD

**Discussion by brianleroux** [architect/architect#1249] [Feature Requests]
should have something like https://github.com/hashicorp/terraform-github-actions for folks that want to use architect w github actions

**Comment by jgallen23** [comment]
We actually have a working version of this for deploying unique staging environments and prod for master merges.  I'll see if we can clean it up to share

**Comment by ryanblock** [comment]
That is dope, I'd love to see it!

**Comment by brianleroux** [comment]
omg yas this is exactly what we wanna build

**Comment by mishra-prabhakar** [comment]
This would be super cool, is it already live and ready to use or what's the status?

**Reply by brianleroux** [reply]
we have examples in the various repos but idk if we've formalized into anything specific yet (cc @macdonst && @ryanblock for any additional context)
